index.html

```diff
@@ -94,6 +94,23 @@ <h3>View3D</h3>
             </p>
         </div>
     </article>
+    <article>
```

Comment:
It would be nice if the InfiniteGrid and Grid had the same location. How about the position like below?
Flicking -> Axes -> InfiniteGrid -> Grid

Reply:
UI components have a high priority.
Axes are used in view360 as well as utility components.

Reply:
I think it's correct. Then, the layout is a little strange, but Flicking -> InfiniteGrid -> Grid -> View360 -> View3d would be good.